Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/rtl/ButtonTopLayers_RTL_LayoutTest.java

```diff
@@ -0,0 +1,75 @@
+package org.catrobat.paintroid.test.espresso.rtl;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.paintroid.MainActivity;
+import org.catrobat.paintroid.R;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Locale;
+
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isClickable;
+import static android.support.test.espresso.matcher.ViewMatchers.isCompletelyDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.catrobat.paintroid.MultilingualActivity.LANGUAGE_TAG_KEY;
+import static org.catrobat.paintroid.test.espresso.util.EspressoUtils.openNavigationDrawer;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+
+@RunWith(AndroidJUnit4.class)
+public class ButtonTopLayers_RTL_LayoutTest {
+    private static final Locale ARABICLOCALE = new Locale("ar");
+
+    @Rule
+    public ActivityTestRule<MainActivity> launchActivityRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Before
+    public void setUp() throws Exception {
+     launchActivityRule.getActivity();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        SharedPreferences sharedPreferences = InstrumentationRegistry.getTargetContext().getSharedPreferences("For_language", Context.MODE_PRIVATE);
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putString(LANGUAGE_TAG_KEY, "en");
```

Comment:
Would it better to just remove the language setting? -> editor.remove(LANGUAGE_KEY)
The default device language of our test devices is usually english

Reply:
you are right, would be better as you wrote